src/scipp/core/__init__.py

```diff
@@ -62,6 +62,11 @@ def custom_formatwarning(msg, *args, **kwargs):
 setattr(GroupByDataArray, 'bins', property(_groupby_bins))
 setattr(GroupByDataset, 'bins', property(_groupby_bins))
 
+from .operations import to as to
+
+setattr(Variable, 'to', to)
+setattr(DataArray, 'to', to)
```

Comment:
Better use `_binding.bind_functions_as_methods` to set up types and meta data properly.

See: https://github.com/scipp/scipp/blob/ed18d03a59c59b25c7bd463d94907668c61d5941/src/scipp/__init__.py#L99-L102